### DISCUSSION TITLE
TDM is keeps buffering

### DISCUSSION THREAD

**Discussion by uttamkorwa** [jziolkowski/tdm#236] [Q&A]
Hi Team,
I'm facing issue in loading tasmota device on my TDM.It keep buffering and not responsive
Please let me how many devices TDM can manage

**Comment by jziolkowski** [comment]
> Hi Team,
> I'm facing issue in loading tasmota device on my TDM.It keep buffering and not responsive
> Please let me how many devices TDM can manage

What do you mean buffering? Tdm doesn't have such feature.

As for the device count I'm aware of a user having more than 100
